### ISSUE TITLE
Can't see shared folder with smb

### ISSUE THREAD

**Issue by valenipes** [zhanghai/MaterialFiles#1153]
When I enable the following smb option in smb.conf, Material Files makes me connect, but doesn't show any file, nor allows me to create any.

server smb encrypt = required

Here the screen of what I see when the when enabled:

![Screenshot_20240312-182911_Material Files](https://github.com/zhanghai/MaterialFiles/assets/162601588/e20e5f12-1f66-4a78-b5b5-2582dd4af1c3)

Here the screen of what I see when the when I remove that line and restart the server:
![Screenshot_20240312-182958_Material Files](https://github.com/zhanghai/MaterialFiles/assets/162601588/7e57703a-c0d2-41d1-bbbd-f8697f74e6b3)
And of course lets me to read and write files.

I think the issue is with Material Files, because I tried to access from Dolphin (debian) with success, but any other android applications didn't work.

**Comment by zhanghai** [comment]
If any other android applications didn't work, it seems to me that's not a problem only with this app?

**Comment by valenipes** [comment]
Yes, probably it's a more general problem. I don't know what causes it, because the server was up and running and again, as a proof of that it worked properly with Dolphin file manager. Maybe is something that isn't available yet with android?

p.s. I'm sorry if I closed and reopened the issue, I'm a bit clumsy
